Dockerfile

```diff
@@ -1,19 +1,15 @@
 FROM node:0.10-slim
 MAINTAINER matthias.luebken@gmail.com
-WORKDIR /app
 
-# install dependencies
-ADD package.json /app/
-RUN npm install
+LABEL Version="1.0.0"
```

Comment:
Background info can be found at
https://github.com/projectatomic/ContainerApplicationGenericLabels and
there is no relation to tags, even thought this relation can be made by a
build system like https://github.com/projectatomic/atomic-reactor

Am Samstag, 16. April 2016 schrieb Matthias Lübken :

> In Dockerfile
> <https://github.com/luebken/currentweather/pull/5#discussion_r59964624>:
>
> >
> > -# install dependencies
> > -ADD package.json /app/
> > -RUN npm install
> > +LABEL Version="1.0.0"
>
> I like it. Do you have more information on this convention. E.g. how this
> relates to the tag?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/luebken/currentweather/pull/5/files/0207e4020887de310e821bb406ee9db7443ea360..4dfff22f5f3b23deb5e19ac21a8deaa96fc56d46#r59964624>
>


-- 
Principal Software Engineer - Systems Design & Engineering
Mobile: +49 171 2801345

Red Hat GmbH, http://www.de.redhat.com/, Sitz: Grasbrunn,
Handelsregister: Amtsgericht München, HRB 153243,
Geschäftsführer: Paul Argiry, Charles Cachera, Michael Cunningham, Michael
O'Neill